upload/styles/cb_28/layout/signup.html

```diff
@@ -168,7 +167,7 @@
 	</div>
 {/if}
 
-<script src="//code.jquery.com/ui/1.10.4/jquery-ui.js"></script>
+<script src="http://code.jquery.com/ui/1.10.4/jquery-ui.js"></script>

```

Comment:
External files should be loaded depending of the current website protocol
If the lib is loader like this from a HTTPS website, it will generate browser alerts in console and the HTTPS lock next to the url will display errors